scripts/tags.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# credit: https://github.com/cli/cli/blob/trunk/script/changelog
+
+function previous_tag() {
+  current_tag="$(git describe --tags HEAD^ --abbrev=0)"
```

Comment:
As we don't run this in CI on tag builds, I don't know if this is even required. I.e. all this function needs to do is get the previous tag which isn't a prerelease.

Reply:
Deleting :)